composer.json

```diff
@@ -16,48 +16,45 @@
     ],
     "require": {
         "php":                      ">=8.1",
-        "symfony/framework-bundle": "^5.4|^6.0",
-        "symfony/http-kernel":      "^5.4|^6.0",
-        "symfony/security-bundle":  "^5.4|^6.0",
-        "symfony/serializer":       "^5.4|^6.0",
-        "symfony/form":             "^5.4|^6.0",
-        "symfony/twig-bundle":      "^5.4|^6.0",
-        "symfony/process":          "^5.4|^6.0",
+        "symfony/framework-bundle": "^5.4|^6.4|^7.0",
+        "symfony/http-kernel":      "^5.4|^6.4|^7.0",
+        "symfony/security-bundle":  "^5.4|^6.4|^7.0",
+        "symfony/serializer":       "^5.4|^6.4|^7.0",
+        "symfony/form":             "^5.4|^6.4|^7.0",
+        "symfony/twig-bundle":      "^5.4|^6.4|^7.0",
+        "symfony/process":          "^5.4|^6.4|^7.0",
         "symfony/ux-turbo": "^2.0",
         "symfony/stimulus-bundle": "^2.0",
-        "symfony/webpack-encore-bundle": "^1.14|^2.0",
-        "araise/core-bundle":     "^1.0",
-        "araise/table-bundle":    "^1.0",
-        "araise/search-bundle":    "^3.0",
+        "symfony/webpack-encore-bundle": "^1.14|^2.1",
+        "araise/core-bundle":     "dev-feature/update-deps",
+        "araise/table-bundle":    "dev-feature/update-deps",
+        "araise/search-bundle":    "dev-feature/update-deps",
         "knplabs/knp-menu-bundle":  "^v3.2.0",
         "doctrine/inflector": "^2.0",
         "symfony/maker-bundle": "^1.0.0",
         "twig/twig": "^3.4.3|^2.15.3",
-        "doctrine/orm": "^2.13.3"
+        "doctrine/orm": "^2.13.3|^3.1",
+        "doctrine/annotations": "^1.13.2|^2.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^5.4|^6.0",
-        "symfony/config": "^5.4|^6.0",
-        "symfony/dependency-injection": "^5.4|^6.0",
-        "symfony/yaml": "^5.4|^6.0",
-        "symfony/validator": "^5.4|^6.0",
-        "nyholm/symfony-bundle-test": "^1.8",
+        "symfony/phpunit-bridge": "^5.4|^6.4|^7.0",
+        "symfony/config": "^5.4|^6.4|^7.0",
+        "symfony/dependency-injection": "^5.4|^6.4|^7.0",
+        "symfony/yaml": "^5.4|^6.4|^7.0",
+        "symfony/validator": "^5.4|^6.4|^7.0",
         "doctrine/doctrine-bundle": "^2.5",
-        "doctrine/annotations": "^1.13.2",
         "whatwedo/php-coding-standard": "^1.0",
         "zenstruck/foundry": "^1.16",
         "zenstruck/console-test": "^v1.1.0",
-        "symfony/orm-pack": "^1.0",
-        "symfony/translation": "^5.4|^6.0",
+        "symfony/translation": "^5.4|^6.4|^7.0",
         "gedmo/doctrine-extensions": "^3.5",
-        "symfony/dotenv": "^5.4|^6.0",
-        "symfony/test-pack": "^1.0",
-        "symfony/runtime": "^5.4|^6.0",
+        "doctrine/orm": "^2.13.3",
```

Comment:
```suggestion
        "doctrine/orm": "^2.13.3|^3.1",
```

Reply:
@rugbymauri we cannot update the requirement-dev to ^3.1 as the DoctrineExtension are not supported yet https://github.com/doctrine-extensions/DoctrineExtensions/issues/2778